### PR TITLE
editorial review

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by renetapopova** [comment]
I don't know how to build it locally as it is not part of the docs. It is published here - https://neo4j.com/graphgists/importing-csv-files-with-cypher/

File: cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc [hunk 1]
```diff
@@ -42,18 +42,19 @@ Nodes created: 5
...
-This time, we're also creating a relationship to the country in which the movie was made.
-If you are storing your data in a SQL database, this is the one-to-many relationship type.
+Now import the movies.
+This time, you will also create a relationship to the country in which the movie was made.
+If you are storing your data in a SQL database, this is a one-to-many relationship type.
 
-We're using `MERGE` to create nodes that represent countries.
-Using `MERGE` avoids creating duplicate country nodes in the case where multiple movies have been made in the same country.
+Use `MERGE` to create nodes that represent countries.
+This avoids creating duplicate country nodes in case multiple movies have been made in a same country.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc]:
```suggestion
This avoids creating duplicate country nodes when multiple movies were made in the same country.
```

**Review by renetapopova — APPROVED** [review]
Just a minor comment. Approving it, but have no idea if that will be published.

**Comment by martin-neotech** [comment]
https://neo4j.com/graphgists/importing-csv-files-with-cypher/

**Pull request merged**